gradleops/AndroidSeed/tealeaf/src/main/jni/gen/js_timestep_view_template.gen.cpp

```diff
@@ -25,28 +25,26 @@
 
```

Comment:
most of the changes in this files are wrong indentation. We can remove all the changes I guess.

Reply:
This file is automatically generated by barista. I will set the barista up to generate files with correct identation.